### PR TITLE
Add browser tests for CiviForm admins creating/editing/deleting statuses

### PR DESCRIPTION
**Pull request by clouser2**
### Description

Adds the necessary browser test infrastructure and associated tests for validating the CiviForm admin experience for working with program statuses. Still to come are server-side and browser-test-side tests for managing translations.

## Release notes:

Add browser tests for program statuses.

### Checklist

- [X] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [X] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

#1572

### CODE REVIEW

File: browser-test/src/support/admin_ti_groups.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {Page} from 'playwright'
 
 export class AdminTIGroups {
-  public page!: Page
+  private page!: Page
```
Comment (clouser2) [inline browser-test/src/support/admin_ti_groups.ts:4]: Technically unrelated, but doesn't hurt anything to add :).

File: browser-test/src/civiform_admin_program_statuses.test.ts [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  beforeAll(async () => {
+    const {page} = await startSession()
+    pageObject = page
+    adminPrograms = new AdminPrograms(pageObject)
+    adminProgramStatuses = new AdminProgramStatuses(pageObject)
+
+    await loginAsAdmin(pageObject)
+  })
+
+  describe('initial status list', () => {
```
Comment (shanemc-goog) [inline browser-test/src/civiform_admin_program_statuses.test.ts:25]: nit: the phrasing is confusing me a little....

maybe: "New  program creation"

File: browser-test/src/civiform_admin_program_statuses.test.ts [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+      })
+      await adminProgramStatuses.expectProgramManageStatusesPage(programName)
+      await adminProgramStatuses.expectStatusExists({
+        statusName: 'Updated status name',
+        expectEmailExists: false,
+      })
+      await adminProgramStatuses.expectStatusNotExists(secondStatusName)
+    })
+
+    it('edits an existing status and configures email', async () => {
```
Comment (shanemc-goog) [inline browser-test/src/civiform_admin_program_statuses.test.ts:142]: edit and delete email would be good cases too.

**Review by shanemc-goog — APPROVED** [review]
Just the two suggestions

File: browser-test/src/civiform_admin_program_statuses.test.ts (hunk 2 above)
Comment (clouser2) [reply browser-test/src/civiform_admin_program_statuses.test.ts]: Done!

File: browser-test/src/civiform_admin_program_statuses.test.ts (hunk 3 above)
Comment (clouser2) [reply browser-test/src/civiform_admin_program_statuses.test.ts]: Good call. Added to this test.

**Review by shanemc-goog — APPROVED** [review]

**Pull request merged**